IBPSA/Fluid/Sources/Examples/Outside_WeatherBus.mo

```diff
@@ -0,0 +1,109 @@
+within IBPSA.Fluid.Sources.Examples;
+model Outside_WeatherBus
+  "Test model for source (sink) with weather bus"
+  import IBPSA;
```

Comment:
Please remove the `import` statement. Dymola adds them automatically but they are not needed.